todo_app/tests/integration/test_app_routes.py

```diff
@@ -0,0 +1,135 @@
+import os
```

Comment:
Minor, but personally I prefer tests to live at the top level of the project (often to mirror the source code structure when we have lots of unit tests) rather than under the todo_app folder. 

Separately, I'd advise adding an empty `__init__.py` file to each of these folders; Python is a little inconsistent about when it requires them but you may well find missing them bites you later when you move the tests inside Docker!

Reply:
Thanks for the heads-up, will do that!